cypress/support/script_loader.js

```diff
@@ -156,11 +156,6 @@ if (Cypress.spec.relative.startsWith('cypress/integration/integration_tests')) {
  * To get around this, keep all such statements within functions that are called
  * at runtime.
  * @param {string} scriptUrl
```

Comment:
why removed?

Reply:
Was a comment fragment left over from a previous version of this function: this text is supposed to be talking about a "callback" parameter that is no longer present.